3rdparty/CMakeLists.txt

```diff
@@ -93,6 +93,21 @@ ExternalProject_Add(libconfig
         CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DBUILD_SHARED_LIBS=OFF
         )
 
+################################################################################
+# Install CJSON
+
+set(CJSON_PREFIX_DIR "${3RD_PARTY_PREFIX_DIR}/cjson")
+set(CJSON_INCLUDE_DIR ${CJSON_PREFIX_DIR}/include)
+set(CJSON_LIBRARY_DIR ${CJSON_PREFIX_DIR}/lib)
+
+ExternalProject_Add(cjson
+        GIT_REPOSITORY https://github.com/DaveGamble/cJSON.git
+        GIT_TAG master
+        PREFIX ${3RD_PARTY_PREFIX_DIR}/cjson
+        INSTALL_DIR ${CJSON_PREFIX_DIR}/install
+        CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DBUILD_SHARED_LIBS=OFF
+        )
```

Comment:
This `ExternalProject_Add` is needed for the DPDK because it has very specific way of building it.
However, some of the other libraries may have better integration with CMake and thus they can be added with just [FetchContent](https://cmake.org/cmake/help/latest/module/FetchContent.html).
I mean, `ExternalProject_Add` will work but somebody who knows CMake a bit better may start to ask question why it's done in this way instead of more common way.